torchgeo/trainers/utils.py

```diff
@@ -54,6 +54,25 @@ def extract_backbone(path: str) -> Tuple[str, "OrderedDict[str, Tensor]"]:
     return name, state_dict
 
 
+def get_input_layer_name_and_module(model: Module) -> Tuple[str, Module]:
+    """Retrieve the input layer name and module from a timm model.
+
+    Args:
+        model: timm model
+
+    .. versionadded: 0.4.1
```

Comment:
Patch releases should not include any new features. I would prefix this with underscore and keep it private for now.